src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Runtime/CSharpTestForEmptyStringsUsingStringLength.Fixer.cs

```diff
@@ -37,5 +37,21 @@ protected override SyntaxNode GetRightOperand(SyntaxNode binaryExpressionSyntax)
         {
             return ((BinaryExpressionSyntax)binaryExpressionSyntax).Right;
         }
+
+        protected override bool IsFixableBinaryExpression(SyntaxNode node)
```

Comment:
You could use expression body.

Reply:
➡️ Marking resolved per https://github.com/dotnet/roslyn-analyzers/pull/1111#discussion_r548020530